### DISCUSSION TITLE
Getting "copilot/1.31.6194", is out of date issue

### DISCUSSION THREAD

**Discussion by hariguru** [community/community#43849] [Copilot Conversations]
### "copilot/1.31.6194", is out of date.

Bug

### Body

I subscribed copilot today and installed the vscode extension. it is not getting activated. Getting the following error in the log
Your editor's Copilot extension, version "copilot/1.31.6194", is out of date. Please install a newer version to access this service.
My VS Code Version
Version: 1.67.2 (system setup)

**Comment by iamcjwilliams** [comment]
@hariguru Sorry for the trouble here, please visit this doc to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-github-copilot-in-visual-studio-code

**Comment by TanveerAliS** [comment]
@iamcjwilliams I'm also facing same issue

`
[INFO] [default] [2023-04-30T19:56:07.542Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2023-04-30T19:56:07.686Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2023-04-30T19:56:07.810Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2023-04-30T19:56:07.835Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 25 ms
[INFO] [fetch] [2023-04-30T19:56:07.836Z] Your editor's Copilot extension, version "copilot/1.31.6194", is out of date. Please install a newer version to access this service.

[INFO] [default] [2023-04-30T19:56:08.377Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2023-04-30T19:56:08.402Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 25 ms
[INFO] [fetch] [2023-04-30T19:56:08.404Z] Your editor's Copilot extension, version "copilot/1.31.6194", is out of date. Please install a newer version to access this service.`

**Reply by davecheney** [reply]
@TanveerAliS that version of copilot is over a year old, you need to upgrade. You probably need to also update VSCode to make the updated extension available.
